### PR TITLE
fix(core): use timestamp if neither localized or universal are set

### PR DESCRIPTION
**Pull request by renaudAmsellem**
## Checklist

Definition Of Review

-   [ ] Own code review done (add notes for others)
-   [ ] Write message in teams channel
    ```
     <Title>

    *️⃣ Impacted projects : Core - DataStudio - Admin - @leav/ui - @leav/utils - ...

    📖 Ticket: https://aristid.atlassian.net/browse/<JIRA_TICKET_IDENTIFIER>

    🧑‍💻 PR: <link to PR/MR>

    ℹ Info: <brief explanation - context - how to test>
    ```

Definition Of Mergeable

-   [ ] 2 approves
-   [ ] 1 functional review - US has been tested
-   [ ] Every comment is handled - blocking ones have been resolved by reviewer
-   [ ] Design OK
-   [ ] Can be tested by POs
-   [ ] PR was introduced during daily meeting

### CODE REVIEW

File: apps/core/src/domain/actions/formatDateAction.spec.ts [hunk 1]
```diff
@@ -97,10 +97,9 @@ describe('formatDateAction', () => {
     });
 
     describe('edge cases', () => {
-        test('should fallback to empty localized param when neither params provided', async () => {
+        test('should fallback to timestamp when neither params provided', async () => {
```
Comment (evoiron) [inline apps/core/src/domain/actions/formatDateAction.spec.ts:100]: **suggestion:** 
neither ne s'utilise pas seul, mais en conjonction avec nor (exemple: neither localized nor universal)
```suggestion
        test('should fallback to timestamp when no params provided', async () => {
```

File: apps/core/src/domain/actions/formatDateRangeAction.spec.ts [hunk 2]
```diff
@@ -84,6 +84,11 @@ describe('formatDateRangeAction', () => {
     });
 
     describe('edge cases', () => {
+        test('should fallback to timestamp when neither params provided', async () => {
```
Comment (evoiron) [inline apps/core/src/domain/actions/formatDateRangeAction.spec.ts:87]: **suggestion:** 
```suggestion
        test('should fallback to timestamp when no params provided', async () => {
```

**Review by evoiron — APPROVED** [review]

**Review by philippechevieux — APPROVED** [review]
